### DISCUSSION TITLE
Browser protocol handler

### DISCUSSION THREAD

**Discussion by SiteRelEnby** [ente-io/ente#4300] [Enhancements]
![image](https://github.com/user-attachments/assets/bddc7903-8b78-4c24-a319-b17350a9c382)
ente auth should install a protocol handler so links like  the lower can be clicked on desktop directly from the browser. 

Seems not to be supported on 4.1.0, tried the usual 3 browsers (firefox/chrome/edge).

Had no way to continue a smooth workflow and had to pick up my phone to do the QR to add it.

**Comment by ua741** [comment]
Duplicate of https://github.com/ente-io/ente/discussions/1285
